modules/persistent-storage-csi-gcp-filestore-nfs-export-options.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_csi-google_cloud_file.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-gcp-filestore-nfs-export-options_{context}"]
+= NFS export options
+
+Network File System (NFS) sets file share permissions when creating a storage class. Using NFS export options propagate these options to the server for the Filestore instance.
```

Comment:
NFS sets permissions when creating a SC?
Maybe ? The CSI driver sets file share permissions when creating volumes?

Reply:
i don't see the change

Reply:
@gcharot Line 9 paragraph was changed.